recipes/nr.bundler/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nr.bundler" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f2217a0f1f2041ae3406d36a984ef14cc848aec6277a2b654235b7c40ea0cf5e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - appdirs >=1.4.3
+    - distlib >=0.2.7
+    - nr.fs >=1.3.1
+    - nr.types >=2.0.0
+    - pefile >=2017.11.5
+    - pip
```

Comment:
I think everything but pip and python can be removed from the host dependency

Reply:
Correct. Can setuptools also be removed? The setup script actually imports it. I built meta.yaml with `conda skeleton` and it put all these deps in the `host` section